Cargo.toml

```diff
@@ -61,7 +61,7 @@ clap = { version = "4", features = ["derive", "env"] }
 clap_mangen = "0.2.2"
 console = { version = "0.15", default-features = false }
 console-subscriber = "0.1.7"
-criterion = "0.4"
+criterion2 = "0.7.0"
```

Comment:
why criterion2 vs criterion?

Reply:
criterion no longer seems to be actively  maintained